ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -24,5 +24,19 @@ file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
 )
 if(_source_files)
   message(STATUS "Added test 'cppcheck' to perform static code analysis on C / C++ code")
-  ament_cppcheck()
+
+  # Get include paths for added targets
+  set(_all_include_dirs "")
+  # BUILDSYSTEM_TARGETS only supported in CMake >= 3.7
+  if(CMAKE_VERSION VERSION_EQUAL "3.7.0" OR CMAKE_VERSION VERSION_GREATER "3.7.0")
```

Comment:
Instead of two conditions repeating the version please use only one condition: `NOT CMAKE_VERSION VERSION_LESS "3.7.0"`